lib/entities.ts

```diff
@@ -1290,6 +1199,98 @@ export type NonTradeActivityActivityType =
    */
   'SSP';
 
+export interface RawTradeActivity {
+  // Only FILL
+  activity_type: Extract<ActivityType, 'FILL'>
```

Comment:
I shuffled some stuff around in here so that TypeScript call help us with `TradeActivity` vs `NonTradeActivity`.

Now you can do stuff like this:

```
function doSomethingWithActivity(activity: TradeActivity | NonTradeActivity) {
  if (activity.activity_type === 'FILL'
    // TypeScript knows we're dealing with a TradingActivity here
  } else {
    // TypeScript knows we're dealing with a NonTradingActivity here
  }
}
```